src/main/java/prefs/UserConfig.java

```diff
@@ -0,0 +1,19 @@
+package prefs;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Test class to see if the file handler works with other config subclasses in the new framework
+ */
+public class UserConfig implements Config {
```

Comment:
@dariusf Seems out of scope of this issue, which is to simply generalise the original file handler. Maybe in another issue to actually create the user prefs? 

My file here is just a dummy file that I intended to delete before merging, just to test/show that the generalisation works (since we don't have more than one functional config file at this point). I'm moving the dummy class to the unit test as per @HansNewbie suggestion